### PR TITLE
Site title color and font size

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-color.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/set-customize';
+describe( 'Site Title color settings in the customizer', () => {
+	it( 'site title color should apply corectly', async () => {
+		const sitetitleColor = {
+            'header-color-site-title': 'rgb(226, 21, 21)',
+            'font-size-site-title': {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-color.test.js:7]: This testcase is related to site title color, font related settings should not be in this file.

File: tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-color.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+				desktop: 72,
+                'desktop-unit': 'px',
+				
+            },
+        };
+
+        await setCustomize( sitetitleColor);
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-color.test.js:14]: Code indentation should be proper.

File: tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-color.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+        await expect( {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-color.test.js:19]: await page.waitForSelector(); is missing

File: tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-typography.test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+describe( 'Site Title Typography settings and color settings in the customizer', () => {
+	it( 'site title typography and color should apply corectly', async () => {
+		const sitetitleTypography = {
+            'body-font-family': 'Great Vibes',
+
+			'body-text-transform': 'lowercase',
+			'font-size-site-title': {
+				desktop: 72,
+                'desktop-unit': 'px',
+            },
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/header-builder/Site-identity-test/Site-title/site-title-typography.test.js:13]: Above array elements are not indentated properly.

**Comment by SimranChougule1** [comment]
I wrote these test cases for practice purpose that's why I'm closing this PR

**Pull request closed without merging**